dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -18,23 +18,25 @@ struct Clip: Hashable {
     let thumbnail: UIImage
     let isHighlighted: Bool
     let metadata: ClipMetadata
+    let showCode: Bool
     
-    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata) {
+    init(url: URL, thumbnail: UIImage, isHighlighted: Bool, metadata: ClipMetadata, showCode: Bool) {
         self.url = url
         self.thumbnail = thumbnail
         self.isHighlighted = isHighlighted
         self.metadata = metadata
+        self.showCode = showCode
```

Comment:
The addition of the `showCode` property to the `Clip` struct is a good enhancement, as it allows for more flexibility in how the clips are displayed. This change will enable the UI to show or hide the code metadata as needed.